toolkits/springer/packages/springer-dropdown/js/dropdown.js

```diff
@@ -11,7 +11,7 @@ const selector = {
 const defaultOptions = {
 	BUTTON_CLASS: '',
 	CLICK_OUTSIDE: true,
-	DROPDOWN_CLASS: '',
+	DROPDOWN_CLASSES: [''],
```

Comment:
Should this default to an empty array? This will have a length of 1.

Reply:
Update: see https://github.com/springernature/frontend-toolkits/pull/191/files#r391114098

Reply:
Ah yes good point